src/api/guardian_filter.js

```diff
@@ -1 +1,13 @@
-module.exports = response => {};
+module.exports = response => {
+  let results = response.results.reduce((articles, article) => {
+    articles.push({
+      id: article.id,
```

Comment:
instead of `id: article.id`  you could do just `id` now

Reply:
done